Doc/whatsnew/2.4.rst

```diff
@@ -1453,7 +1453,7 @@ Some of the changes to Python's build process and to the C API are:
   extension functions: :c:macro:`Py_RETURN_NONE`, :c:macro:`Py_RETURN_TRUE`, and
   :c:macro:`Py_RETURN_FALSE`. (Contributed by Brett Cannon.)
 
-* Another new macro, :c:macro:`Py_CLEAR(obj)`,  decreases the reference count of
+* Another new macro, :c:expr:`Py_CLEAR(obj)`,  decreases the reference count of
```

Comment:
I don't understand this change, as the `:c:macro:` invocations above were left alone. Looks like the difference is simply that here we are "calling" the `Py_CLEAR`? Maybe instead of making it an expression, we should remove the `(obj)` part and leave it as `Py_CLEAR`?

Reply:
```suggestion
* Another new macro, :c:macro:`Py_CLEAR`,  decreases the reference count of
```

Reply:
Makes sense, done.

A